website/docs/docs/cloud/dbt-cloud-ide/lint-format.md

```diff
@@ -0,0 +1,189 @@
+
+To format your SQL code, dbt Cloud integrates with [sqlfmt](http://sqlfmt.com/), which is an uncompromising SQL query formatter that provides one way to format the SQL query and Jinja.
+
+To enable formatting:
+
+1. Make sure you open a `.sql` file.
+2. Click on the **`</> Config`** button on the right side of the console. (Make sure you're in a .sql file).
+3. In the code quality tool config pop up, you have the option to select sqlfluff or sqlfmt. 
+4. To format your code, select the **sqlfmt** radio button. (Use sqlfluff to [lint](#linting) your code).
+5. Once you've selected the **sqlfmt** radio button, go to the console section (located below the **File editor**) to select the **Format** button. 
+6. The **Format** button auto-formats your code in the **File editor**. Once you've auto-formatted, you'll see a message confirming the outcome. 
+
+<Lightbox src="/img/docs/dbt-cloud/cloud-ide/sqlfmt.gif" width="95%" title="Use sqlfmt to format your SQL code."/>
+
+### Format YAML, Markdown, JSON
+
+To format your YAML, Markdown, or JSON code, dbt Cloud integrates with [Prettier](https://prettier.io/), which is an opinionated code formatter.
+
+1. To enable formatting, make sure you open a `.yml`, `.md`, or `.json` file.
+2. In the console section (located below the **File editor**), select the **Format** button to auto-format your code in the **File editor**. 
+3. Once you've auto-formatted, you'll see a message confirming the outcome. 
+
+<Lightbox src="/img/docs/dbt-cloud/cloud-ide/prettier.gif" width="95%" title="Format YAML, Markdown, and JSON files using Prettier."/>
+
+### Format Python
+
+To format your Python code, dbt Cloud integrates with [Black](https://black.readthedocs.io/en/latest/), which is an uncompromising Python code formatter.
+
+1. To enable formatting, make sure you open a `.py` file.
+2. In the console section (located below the **File editor**), select the **Format** button to auto-format your code in the **File editor**. 
+3. Once you've auto-formatted, you'll see a message confirming the outcome. 
+
+## Troubleshooting
+
+<details>
+<summary>Can I nest <code>.sqlfluff</code> files?</summary>
+
+To ensure optimal code quality, it's highly recommended you have one main `.sqlfluff` configuration file in the root folder of your project. However, you have the flexibility to customize and include an additional child `.sqlfluff` configuration file within specific subfolders of your dbt project. <br /><br />By nesting a `.sqlfluff` file in a subfolder, SQLFluff will apply the rules defined in that subfolder's configuration file to any files located within it. For all other files and folders outside of the subfolder, the rules specified in the parent `.sqlfluff` file will be used. This hierarchical approach allows for tailored linting rules while maintaining consistency throughout your project. Refer to [SQLFluff documentation](https://docs.sqlfluff.com/en/stable/configuration.html#configuration-files) for more info.
```

Comment:
We might want to include the downsides of having child files if we're going to use strong language recommending against it.

Reply:
@bossnunta @dave-connors-3 can you provide what the downsides could possible be to address Matt's question?

Reply:
i think we can take a strong stance that multiple files leaves the door open to multiple sql styles and therefore inconsistent code.